importexport/musicxml/musicxmlsupport.cpp

```diff
@@ -520,6 +524,10 @@ AccidentalType mxmlString2accidentalType(const QString mxmlName)
       map["natural-up"] = AccidentalType::NATURAL_ARROW_UP;
       map["flat-down"] = AccidentalType::FLAT_ARROW_DOWN;
       map["flat-up"] = AccidentalType::FLAT_ARROW_UP;
+      map["double-sharp-down"] = AccidentalType::SHARP2_ARROW_DOWN;
+      map["double-sharp-up"] = AccidentalType::SHARP2_ARROW_UP;
+      map["flat-flat-down"] = AccidentalType::FLAT2_ARROW_DOWN;
+      map["flat-flat-up"] = AccidentalType::FLAT2_ARROW_UP;
```

Comment:
not sure though how or whether to handle an import of "other".

Reply:
As mentioned above, handling `"other"` would only make sense, if we'd check the `@smufl` attribute. So, for now, I think it's ok to ignore `"other"`.